src/main/java/core/basesyntax/TestForTheTask.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.FruitTransactionDaoImpl;
+import core.basesyntax.service.TransactionStrategy;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class TestForTheTask {
+
+    public static void main(String[] args) {
+        TransactionStrategy transactionStrategy = new TransactionStrategy();
+        FruitTransactionDaoImpl fruitTransactionDao = new FruitTransactionDaoImpl();
+        fruitTransactionDao.addToStorage("data.CSV");
+        transactionStrategy.getShopService();
+        fruitTransactionDao.getReport();
```

Comment:
accurately ensure that your main method looks exactly like algorithm from here:
![image](https://github.com/mate-academy/jv-fruit-shop/assets/73041091/9c32f3f0-68ae-4821-b0ba-c58051a42706)
for the moment, its invalid